resources/views/applicant/strategic_response_application/common/skill-form.html.twig

```diff
@@ -79,7 +79,7 @@
         {% if locale == "application" and application_step == "4" and criterion == false %}template{% endif %}
         {% if skill_declaration %}complete{% endif %}
         modal-target-object"
-    data-item-saved="{{ skill_declaration ? "true" : "" }}" data-item-url="{{ route("skill_declarations.destroy", ":id") }}" data-item-id="{{ skill_declaration ? skill_declaration.id : 0 }}">
+    data-item-saved="{{ skill_declaration ? "true" : "" }}" data-item-url="" data-item-id="{{ skill_declaration ? skill_declaration.id : 0 }}">
```

Comment:
I believe we can remove this entire folder `resources/views/applicant/strategic_response_application`. Since it's all been moved to its own app.

Reply:
Created an issue https://github.com/GCTC-NTGC/TalentCloud/issues/4701

Reply:
Nice, thanks 👍